dev/io.openliberty.microprofile.metrics.internal.public/.project

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
-	<name>com.ibm.ws.microprofile.metrics.3.x.monitor_fat</name>
+	<name>io.openliberty.microprofile.metrics.internal.public</name>
```

Comment:
Not sure this is a git problem or the file is wrongly renamed.

Reply:
Right, should be a git issue.
Projects were copied/duplicated then renamed.
Then the original `com.ibm.*` projects were deleted at last.